server/database/models.js

```diff
@@ -1,8 +1,14 @@
-let fetch = (model, callback) => {
```

Comment:
Maybe consider naming this something else since fetch is already a native javascript method, just to avoid potential confusion